frontend/build.gradle

```diff
@@ -2,26 +2,21 @@ plugins {
     id 'java'
     id 'org.springframework.boot' version '2.2.5.RELEASE'
     id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-
 }
-
 version 'unspecified'
-
 sourceCompatibility = 1.11
-
 repositories {
     mavenCentral()
 }
-
 dependencies {
-
     testCompile group: 'junit', name: 'junit', version: '4.12'
-    implementation 'org.springframework.boot:spring-boot-starter-web'
     compile project(':domain')
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
         implementation 'com.google.code.gson:gson:2.8.6'
-
+        compile 'org.springframework.boot:spring-boot-starter-web:2.0.1.RELEASE'
+       compile group: 'com.h2database', name: 'h2', version: '1.3.148'
+        compile group: 'org.hibernate.javax.persistence', name: 'hibernate-jpa-2.1-api', version: '1.0.0.Final'
```

Comment:
We probably do not need these dependencies (h2 and hibernate) at this stage since we are still using in-memory data structures to hold the data. We can add them later when we add the database service.

Reply:
resolved